README.md

```diff
@@ -1,6 +1,6 @@
-# AutoRA Uncertainty Sampler
+# AutoRA Uncertainty Experimentalist
 
-The uncertainty sampler identifies experimental conditions $\vec{x}' \in X'$ with respect model uncertainty. Within the uncertainty sampler, there are three methods to determine uncertainty:
+The uncertainty experimentalist identifies experimental conditions $\vec{x}' \in X'$ with respect model uncertainty. Within the uncertainty sampler, there are three methods to determine uncertainty:
```

Comment:
looks like we need to change 'sampler' to 'experimentalist' in second sentence also, correct? also missing 'to' in 'with respect to model uncertainty'